cucumber/api/features/support/env.rb

```diff
@@ -5,7 +5,7 @@
 ENV['CONJUR_LOG_LEVEL'] ||= 'debug'
 
 parallel_cuke_vars = {}
-parallel_cuke_vars['CONJUR_APPLIANCE_URL'] = "http://conjur#{ENV['TEST_ENV_NUMBER']}"
+parallel_cuke_vars['CONJUR_APPLIANCE_URL'] = ENV.fetch('CONJUR_APPLIANCE_URL', "http://conjur#{ENV['TEST_ENV_NUMBER']}")
```

Comment:
This enabled running cucumber tests in the dev env, where you [run them from the Conjur container](https://github.com/cyberark/conjur/blob/master/CONTRIBUTING.md#cucumber). In that case, you need to use `http://localhost:3000` instead of `http://conjur`.